docs/index/create-daml-apps/developer-tools/index.rst

```diff
@@ -0,0 +1,14 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+Developer Tools
```

Comment:
```suggestion
.. SPDX-License-Identifier: Apache-2.0

Developer Tools
```
Sphinx cries if there isn't a blank line in-between the comment and the title.

Which makes me a little bit sad.